sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableClient.java

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.data.tables;
+
+import com.azure.core.annotation.ServiceClient;
+
+import java.util.List;
+import java.util.Map;
+
+@ServiceClient(
+    builder = TableClientBuilder.class)
+public class TableClient {
+    String tableName;
+
+    TableClient(String tableName) {
+        this.tableName = tableName;
+    }
+
+    /**
+     * Queries and returns entities in the given table using the select and filter strings
+     *
+     * @param selectString odata select string
+     * @param filterString odata filter string
+     * @return a list of the tables that fit the query
+     */
+    public List<TableEntity> queryEntity(String selectString, String filterString) {
```

Comment:
Should there also be a way to get a single TableEntity by its row & partition key? e.g. `@Nullable public TableEntity getEntity(String rowKey, String partitionKey)`?

Reply:
Did we still want to resolve this comment?